frontend/src/pages/hardwareProfiles/HardwareProfilesTableRow.tsx

```diff
@@ -46,6 +47,7 @@ const HardwareProfilesTableRow: React.FC<HardwareProfilesTableRowProps> = ({
   hardwareProfile,
   rowIndex,
   migrationAction,
+  isMigratedTable,
```

Comment:
you should just be able to use `migrationAction` o know if it is a migrated row as we only pass that migration action in if it is in a migrate table